### PR TITLE
Juan sgw 191 two period characters crash

### PR DESCRIPTION
**Pull request by juanolmedo1**
Ticket: https://constellationnetwork.atlassian.net/browse/SGW-191

# Screenshots
### Before
Simulator Screen Shot - iPhone 11 Pro Max - 2022-03-23 at 09 36 23

### After
If a period is not present, the user is able to enter only one:
Simulator Screen Shot - iPhone 11 Pro Max - 2022-03-23 at 09 34 39

The user is unable to enter a second period:
Simulator Screen Shot - iPhone 11 Pro Max - 2022-03-23 at 09 34 51

### CODE REVIEW

File: source/scenes/home/SendAsset/Send/Send.container.tsx [hunk 1]
```diff
@@ -62,55 +52,44 @@ import useGasEstimate from 'hooks/useGasEstimate';
...
-}) => {
+  navigation: any;
+}
 
+const SendContainer: FC<IWalletSend> = ({ initAddress = '', navigation }) => {
   const accountController = getAccountController();
   let isExternalRequest = false;
 
-  if(!!location){
+  if (location) {
```
Comment (cmirandac665) [inline source/scenes/home/SendAsset/Send/Send.container.tsx:69]: Why was this changed?

Comment (juanolmedo1) [reply source/scenes/home/SendAsset/Send/Send.container.tsx:69]: Prettier made this update. I'll revert it.

Comment (cmirandac665) [reply source/scenes/home/SendAsset/Send/Send.container.tsx:69]: Sounds good.

**Review by cmirandac665 — APPROVED** [review]
